doc/rst/source/gmtbinstats.rst

```diff
@@ -0,0 +1,201 @@
+.. index:: ! gmtbinstats
+.. include:: module_core_purpose.rst_
+
+***********
+gmtbinstats
+***********
+
+|gmtbinstats_purpose|
+
+Synopsis
+--------
+
+.. include:: common_SYN_OPTs.rst_
+
+**gmt gmtbinstats** [ *table* ] |-G|\ *out_grdfile*
+|SYN_OPT-I|
+|-C|\ **a**\|\ **d**\|\ **g**\|\ **i**\|\ **l**\|\ **L**\|\ **m**\|\ **n**\|\ **o**\|\ **p**\|\ **q**\ [*quant*]\|\ **r**\|\ **s**\|\ **u**\|\ **U**\|\ **z**
+|SYN_OPT-R|
+|-S|\ *search_radius*
+[ |-E|\ *empty* ]
+[ |-N| ]
+[ |-T|\ [**h**\|\ **r**] ]
+[ |SYN_OPT-V| ]
+[ |-W|\ [**+s**] ]
+[ |SYN_OPT-bi| ]
+[ |SYN_OPT-di| ]
+[ |SYN_OPT-e| ]
+[ |SYN_OPT-f| ]
+[ |SYN_OPT-g| ]
+[ |SYN_OPT-h| ]
+[ |SYN_OPT-i| ]
+[ |SYN_OPT-n| ]
+[ |SYN_OPT-qi| ]
+[ |SYN_OPT-r| ]
+[ |SYN_OPT-:| ]
+[ |SYN_OPT--| ]
+
+|No-spaces|
+
+Description
+-----------
+
+**gmtbinstats** reads arbitrarily located (x,y[,z][,w]) points
+(2-4 columns) from standard input [or *table*] and for each
+node in the specified grid layout determines which points are
+within the given radius.  These point are then used in the
+calculation of the specified statistic. The results may be
+presented as is or may be normalized by the circle area to
+perhaps give density estimates.  Alternatively, select
+hexagonal tiling instead of a rectangular grid layout.
```

Comment:
```suggestion
presented as is or may be normalized by the circle area to
perhaps give density estimates.  Alternatively, select
hexagonal tiling instead or a rectangular grid layout.
```

Not sure if I understand this sentence correctly. Is the circle area based on a rectangular grid, and there is an alternative hexagonal tiling scheme? Or are there circular 'tiles', hexagonal tiles, and rectangular grid tiles.